.dir-locals.el

```diff
@@ -0,0 +1,3 @@
+((nil . ((indent-tabs-mode . nil)
+         (require-final-newline . t)))
+ (clojure-mode . ((cljr-favor-prefix-notation . nil))))
```

Comment:
[I've been contributing to Cloverage a lot lately](https://github.com/cloverage/cloverage/pulls?q=is%3Apr+is%3Aclosed+author%3Acamsaul) so this is more for my own benefit than anything else. Well anyone who uses Emacs would benefit from the config stuff